packages/api-catalog-service/.env.example

```diff
@@ -0,0 +1,23 @@
+## Uncomment the lines as needed
+
+NAME=local
+NODE_ENV=local
+PRODUCTION=false
+
+## Database
+DB_PATH=mongo
+DB_NAME=api-catalog
+# DB_USER=
+# DB_PASSWORD=
+
+NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
Should not be a mandatory environment variable. This should only be a workaround for local development, and is not recommended.

```suggestion
# NODE_TLS_REJECT_UNAUTHORIZED=0
```

Reply:
updated